DependencyInjection/CompilerPass/FixturesCompilerPass.php

```diff
@@ -21,9 +22,29 @@ public function process(ContainerBuilder $container)
 
         $fixtures = [];
         foreach ($taggedServices as $serviceId => $tags) {
-            $fixtures[] = new Reference($serviceId);
+            $fixtures[] = [
+                'fixture' => new Reference($serviceId),
+                'groups' => $this->getFixtureGroups($serviceId, $container)
+            ];
+
         }
 
         $definition->addMethodCall('addFixtures', [$fixtures]);
     }
+
+    private function getFixtureGroups($service, ContainerBuilder $container) {
```

Comment:
Thanks for the contribution! Can you move this trailing opening curly bracket down to the next line?